src/black/__init__.py

```diff
@@ -959,6 +962,10 @@ def validate_cell(src: str) -> None:
     """
     if any(transformed_magic in src for transformed_magic in TRANSFORMED_MAGICS):
         raise NothingChanged
+    if src[:2] == "%%" and src.split()[0] not in (
+        "%%" + magic for magic in PYTHON_CELL_MAGICS
+    ):
```

Comment:
really minor, but perhaps
```suggestion
    if src[:2] == "%%" and src.split()[0][2:] not in PYTHON_CELL_MAGICS:
```
is simpler / marginally more efficient?

(feel free to disregard this comment of course 😆 )

Reply:
makes perfect sense, thanks for simplifying it!